### ISSUE TITLE
Cant run the tests

### ISSUE THREAD

**Issue by edosyhptra** [vgarvardt/go-oauth2-pg#9]
I've got the go-oauth2-pg as well as the adapter by run the commands below,

```
$ go get -u -v github.com/vgarvardt/go-oauth2-pg
$ go get -u -v github.com/vgarvardt/go-pg-adapter
```

But when I try to run the tests program I had an error with messages below,
`../../vgarvardt/go-pg-adapter/pgxadapter/adapter.go:6:2: cannot find package "github.com/vgarvardt/pgx-helpers" in any
 of:
        /usr/local/Cellar/go/1.13.7/libexec/src/github.com/vgarvardt/pgx-helpers (from $GOROOT)
        /Users/username/go/src/github.com/vgarvardt/pgx-helpers (from $GOPATH)`

Am I missing something? 

Thank you

**Comment by vgarvardt** [comment]
What pg driver are you using - `pq`, `pgx v3` or `pgx v4`?

**Comment by edosyhptra** [comment]
> What pg driver are you using - `pq`, `pgx v3` or `pgx v4`?

I just download it last week. So it should be pgx v4.

**Comment by edosyhptra** [comment]
Is it because of the incompatible version?

**Comment by vgarvardt** [comment]
Yes, the problem is with versions - I just updated `pgx-helpers` to support v4, now need to update other packages as well.

**Comment by edosyhptra** [comment]
Let me know if its available to use to, please.

**Comment by vgarvardt** [comment]
Added pgx v4 support and released `v1.3.0` - https://github.com/vgarvardt/go-oauth2-pg/releases/tag/v1.3.0 - please check if it solves your problem.
